vdev/src/git.rs

```diff
@@ -51,6 +51,24 @@ pub fn list_files() -> Result<Vec<String>> {
         .collect())
 }
 
+// Get a list of files that have been modified, as a vector of strings
+pub fn get_modified_files() -> Result<Vec<String>> {
+    let args = vec![
+        "ls-files",
+        "--full-name",
+        "--modified",
+        "--others",
+        "--exclude-standard",
+    ];
+    Ok(capture_output(&args)?.lines().map(str::to_owned).collect())
+}
+
+// Gets path of the current Git repository's .git directory by the git rev-parse --git-dir command.
+pub fn get_git_dir() -> Result<String> {
+    let output = capture_output(&["rev-parse", "--git-dir"])?;
+    Ok(output.trim_end().to_string())
+}
```

Comment:
I think we can just use `app::path()` plus `".git"` for this.

Reply:
I have implemented app::path() and removed the extra function `get_git_dir()` I created in the git crate